modules/persistent-storage-local-tolerations.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-tolerations_{context}"]
+= Using tolerations with Local Storage Operator Pods
+
+You can control which nodes the Local Storage Operator Pod runs on and prevent
+other workloads from using those nodes by using tolerations on the Pods.
```

Comment:
This isn't entirely correct. You'd prevent workloads from running on nodes by adding taint values to the nodes. You can then schedule Pods to run those nodes by adding Tolerations to the Pod or DaemonSet definition.

I think what we might want to say here is:

> Taints can be applied to nodes to prevent them from running general workloads. To allow the Local Storage Operator to use these tainted nodes you must define tolerations that allow the created resources to run on these nodes.

Reply:
This comment explains taints and tolerations better than anything else I've read: 

> You'd prevent workloads from running on nodes by adding taint values to the nodes. You can then schedule Pods to run those nodes by adding Tolerations to the Pod or DaemonSet definition.

So I combined that with your suggested text.